constraints/miden-vm/range_checker.air

```diff
@@ -0,0 +1,61 @@
+mod RangeCheckerAir
+
+### Helper functions ##############################################################################
+
+# Returns binary negation of the value.
+fn binary_not(e: scalar) -> scalar:
+    return 1 - e
```

Comment:
nit: I know this was here before, but I find it strange to name this e, since e has a meaning and is never used as an indeterminate. I would probably use `v` or `x` instead

Reply:
I think we can simply get rid of this function and use `!` instead.